### PR TITLE
Develop Soden rename tags

### PR DESCRIPTION
**Pull request by SodenNedos**

### CODE REVIEW

File: BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs [hunk 1]
```diff
@@ -19,7 +22,9 @@ public SodenFunctionalProgramingTests()
         }
 
         [TestMethod]
-        public void a_there_are_3_black_vehicles_and_horsePorwer_min100_and_started()
+        [TestCategory("Soden")]
```
Comment (hossmi) [inline BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs]: If you add categorioes at class level, you do not need categories at method level.
Please rename method category.

File: BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs [hunk 2]
```diff
@@ -30,27 +35,34 @@ public void a_there_are_3_black_vehicles_and_horsePorwer_min100_and_started()
         }
 
         [TestMethod]
-        public void b_there_are_two_started_engines_more_one_doors_closed()
+        [TestCategory("Soden")]
+        [TestCategory("Functional Programing")]
```
Comment (hossmi) [inline BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs]: Please rename method category.

File: BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs [hunk 3]
```diff
@@ -30,27 +35,34 @@ public void a_there_are_3_black_vehicles_and_horsePorwer_min100_and_started()
...
                 .getAll()
                 /**/
                 .Select(vehicle => vehicle.Engine);
+
             Assert.AreEqual(2, engines.Count());
         }
 
         [TestMethod]
+        [TestCategory("Soden")]
+        [TestCategory("Functional Programing")]
```
Comment (hossmi) [inline BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs]: Please rename method category.

File: BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs [hunk 4]
```diff
@@ -30,27 +35,34 @@ public void a_there_are_3_black_vehicles_and_horsePorwer_min100_and_started()
...
                /**/
                .Count();
 
-                Assert.AreEqual(6, pressure);
+            Assert.AreEqual(6, pressure);
         }
 
         [TestMethod]
+        [TestCategory("Soden")]
+        [TestCategory("Functional Programing")]
```
Comment (hossmi) [inline BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs]: Please rename method category.

**Comment by alloret1993** [comment]
Hola Pablo, nos hemos quedado sin sala, estamos en la cafetería

File: BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs (hunk 4 above)
Comment (alloret1993) [reply BusinessCore.Tests/FunctionalProgramingTests/SodenFunctionalProgramingTests.cs]: Hola Pablo, nos han quitado la sala, estamos en la Office 1

**Pull request merged**